### PR TITLE
add notification and access tabs to wfjt

### LINKED ISSUE #5902: WFJT Access Related Tab
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
Hook it up, should be like all the other access tabs.  See https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Template/Template.jsx#L180 as an example.  The only difference should be that we need to pass in the workflow job template api instance.

### PR DESCRIPTION
**Pull request by jlmitch5**
fixes #5902 #5904

### CODE REVIEW

File: awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx [hunk 1]
```diff
@@ -67,8 +76,28 @@ class WorkflowJobTemplate extends Component {
...
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'auditor_role',
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'admin_role',
+        }),
+      ]);
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx]: https://github.com/ansible/awx/pull/6185/files#diff-b91ff4e4db2d2c5d939ecbb4b9fc8353R58-R92

(**edit:** multi-line highlight links for PR files seem to be broken lately - I'm referring to lines 58-92)

This method is now too big and we should break it up into smaller function calls.

File: awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx [hunk 2]
```diff
@@ -67,8 +76,28 @@ class WorkflowJobTemplate extends Component {
...
+          role_level: 'notification_admin_role',
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'auditor_role',
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'admin_role',
+        }),
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx]: I don't think we need all three of these requests for WFJTS. On [Template.jsx](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Template/Template.jsx#L53) , it looks  like we only request `notification_admin_role`.

I only see one request to `/api/organizations/...` in the network tab when loading a WFJT and clicking into notifications in the current app.

Comment (mabashian) [reply awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx]: @jlmitch5 I think I included these requests in the Project.jsx as well but I don't think they're needed for any view except Orgs.  We should either rip them out of Project.jsx too or create an issue to do so outside of this PR.

**Review by mabashian — APPROVED** [review]

**Comment by jlmitch5** [comment]
regate

**Review by jakemcdermott — APPROVED** [review]

**Comment by jlmitch5** [comment]
regate

**Pull request merged**